README.md

```diff
@@ -12,17 +12,30 @@ Contains the __@itwin/insights-client__ package that wraps sending requests to t
 
 ## Environment Variables
 
-```
-# ---- Optional URL prefix for dev/qa environments ----
+### Optional URL prefix for dev/qa environments
```

Comment:
Keep all the `.env` file entries in a code block, so it is easier to copy.

Reply:
made a code block